src/app/core/components/archive-settings-dialog/archive-settings-dialog.component.ts

```diff
@@ -0,0 +1,64 @@
+import { Component, OnInit } from '@angular/core';
+import { IsTabbedDialog, DialogRef } from '@root/app/dialog/dialog.module';
+import { ApiService } from '@shared/services/api/api.service';
+import { AccountService } from '@shared/services/account/account.service';
+import { TagsService } from '@core/services/tags/tags.service';
+import { TagVO } from '@models/tag-vo';
+
+type ArchiveSettingsDialogTab = 'manage-tags';
+
+
+@Component({
+  selector: 'pr-archive-settings-dialog',
+  templateUrl: './archive-settings-dialog.component.html',
+  styleUrls: ['./archive-settings-dialog.component.scss']
+})
+export class ArchiveSettingsDialogComponent implements OnInit {
+  public activeTab: ArchiveSettingsDialogTab = 'manage-tags';
+  public tags: TagVO[] = [];
+  public loadingTags: boolean = true;
+  public hasAccess: boolean;
+
+  constructor(
+    private dialogRef: DialogRef,
+    private api: ApiService,
+    private account: AccountService,
+    private tagsService: TagsService,
+  ) { }
+
+  public ngOnInit(): void {
+    const accessRole = this.account.getArchive().accessRole;
+    this.hasAccess = accessRole === "access.role.owner"; /*|| accessRole === "access.role.manager"*/
+    if (this.hasAccess) {
+      this.api.tag.getTagsByArchive(this.account.getArchive()).then((response) => {
+        this.tags = response.getTagVOs();
+        this.bindTagsToArchive();
+        this.loadingTags = false;
+      }).catch(() => {
+        setTimeout(() => {
+         this.ngOnInit()
+       }, 1000);
+     });
+   }
+  }
+
+  public refreshTags(): void {
+    this.tagsService.refreshTags();
+    this.ngOnInit();
+  }
+
+  public onDoneClick(): void {
+    this.dialogRef.close();
+  }
+
+  public setTab(tab: string): void {
+    // do nothing for now
```

Comment:
Yes! Since there's only one tab in the dialog right now, we don't have to do anything when the user clicks on a tab since there are no other ones to change to. I already implemented the function because I mainly copy-pasted the HTML structure from our other dialogs into this new archive settings dialog, including calls to `setTab`.